.github/workflows/actions/deploy-environment/action.yml

```diff
@@ -75,8 +82,46 @@ runs:
         make ci ${{ inputs.environment_name }} terraform-apply
         cd terraform && echo ::set-output name=url::https://$(terraform output -raw auth_server_fqdn)/
         echo ::set-output name=postgres_server_name::$(terraform output -raw postgres_server_name)
+        echo ::set-output name=blue_green::$(terraform output -raw blue_green)
+
+        if [ "$blue_green" = "true" ]; then
+          echo ::set-output name=slot_name::$(terraform output -raw web_app_slot_name)
+        else
+          echo ::set-output name=slot_name::production
+        fi
       env:
         ARM_ACCESS_KEY: ${{ steps.get_secrets.outputs.TFSTATE-CONTAINER-ACCESS-KEY }}
         TF_VAR_azure_sp_credentials_json: ${{ inputs.azure_credentials }}
         IMAGE_TAG: ${{ inputs.image_tag }}
       shell: bash
+
+    - uses: azure/webapps-deploy@v2
+      with:
+        app-name: ${{ steps.config.outputs.resource_prefix }}getanid-${{ inputs.environment_name}}-auths-app
+        images: ghcr.io/dfe-digital/get-an-identity:authserver-${{ github.sha }}
+        slot-name: ${{ steps.terraform.outputs.slot_name }}
+
+    - uses: azure/CLI@v1
+      if: ${{ steps.terraform.outputs.slot_name == 'staging' }}
+      with:
+        inlineScript: |
+          az webapp deployment slot swap  -g ${{ steps.config.outputs.resource_group_name }} -n ${{ steps.config.outputs.resource_prefix }}getanid-${{ inputs.environment_name}}-auths-app --slot ${{ inputs.slot_name }} --target-slot production
```

Comment:
inputs.slot_name will not work as there is no more such input
maybe steps.terraform.outputs... ?